doc/src/guide/listeners.asciidoc

```diff
@@ -171,10 +170,10 @@ file must not exist: Ranch must be able to create it.
 
 [source,erlang]
 {ok, _} = ranch:start_listener(tcp_echo,
-    ranch_tcp, #{socket_opts => [
-        {ip, {local, "/tmp/ranch_echo.sock"}},
-        {port, 0}
-    ]}, echo_protocol, []
+        ranch_tcp, #{socket_opts => [
+                {ip, {local, "/tmp/ranch_echo.sock"}},
+                {port, 0}
+        ]}, echo_protocol, []
```

Comment:
Sounds like these spaces should be converted to tabs instead.

Reply:
Probably... =^^=

Reply:
Hm, actually, no... in other places in the doc sources, (4) spaces are used for indentation. So I will convert tabs to spaces instead here, ok?

Reply:
Right I thought that was weird. Please make sure it's 4 spaces not 8.

Reply:
See last commit.